elastic/integrations
# [Elasticseach] [Kibana] Add host ip mapping

## What does this PR do?

This PR adds `host.ip` mapping to datastreams: 
- Elasticsearch
        - deprecation
        - gc
        - server
- Kibana
        - log


## Checklist


## How to test this PR locally

Follow the guide to add the integration
Go to Kibaga -> Dataviews and check for the `host.ip` field (there shouldn't be any type conflicts)

<img width="1530" alt="image" src="https://user-images.githubusercontent.com/14139027/234870075-722da169-5fba-4646-9d70-24f4cc65f501.png">


## Related issues

- Closes #5788
